src/runtime_es6_classes.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
I think this file should probably be called something like `runtime_exceptions.js`.. its current names makes me think its here to somehow support ES6 classes.

Reply:
4abccbc27077f67b0e0f9c3eb97866d3689ad9ec